spec/controllers/api/debates_controller_spec.rb

```diff
@@ -0,0 +1,36 @@
+require 'rails_helper'
+
+describe Api::DebatesController do
+  before do
+    request.env['HTTP_AUTHORIZATION'] = auth_token.value
+  end
+
+  context 'when debate is opened' do
+    let(:debate) { create(:debate) }
+    let(:auth_token) { debate.auth_tokens.create }
+
+    it 'returns valid debates hash' do
+      get :show
+      json_response = JSON.parse(response.body)
+
+      expect(response.status).to eq(200)
```

Comment:
same as above, `response` can be subject here and `json_response` can be extracted to `let`